### PR TITLE
Feature/merge 3.6 develop

### PR DESCRIPTION
**Pull request by CuriousVini**
Conflicts: 
- Hash in build.sh
- version in json
- AuthServerAnnounceTest#startUp() was creating object of NettyRouter. Changed it to use cConf as well.

Build passes locally. 

Build: http://builds.cask.co/browse/CDAP-DUT4916-2

### CODE REVIEW

File: cdap-ui/server/config/cdap-ui-config.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "tracker": {
     "artifact": {
       "name": "tracker",
-      "version": "0.3.0-SNAPSHOT",
+      "version": "0.2.0",
```
Comment (wolf31o2) [inline cdap-ui/server/config/cdap-ui-config.json]: This shouldn't be changed on `develop`

File: cdap-examples/SparkPageRank/pom.xml [hunk 2]
```diff
@@ -21,7 +21,11 @@
   <parent>
     <artifactId>cdap-examples</artifactId>
     <groupId>co.cask.cdap</groupId>
+<<<<<<< HEAD
     <version>4.0.0-SNAPSHOT</version>
+=======
+    <version>3.6.0</version>
+>>>>>>> release/3.6
```
Comment (wolf31o2) [inline cdap-examples/SparkPageRank/pom.xml]: Conflict which needs resolving.

File: cdap-ui/templates/apps/predefined/AmazonSQSToHBase.json [hunk 3]
```diff
@@ -26,7 +26,7 @@
                     "artifact": {
                         "name": "core-plugins",
                         "scope": "SYSTEM",
-                        "version": "1.5.0-SNAPSHOT"
+                        "version": "1.4.1"
```
Comment (wolf31o2) [inline cdap-ui/templates/apps/predefined/AmazonSQSToHBase.json]: Keep the version from `develop` not `release/3.6`

File: cdap-ui/templates/apps/predefined/AmazonSQSToHBase.json [hunk 4]
```diff
@@ -45,7 +45,7 @@
...
```
Comment (wolf31o2) [inline cdap-ui/templates/apps/predefined/AmazonSQSToHBase.json]: Keep the version from `develop` not `release/3.6`

File: cdap-ui/templates/apps/predefined/EventClassifier.json [hunk 5]
```diff
@@ -44,7 +44,7 @@
                     "label": "Events",
                     "artifact": {
                         "name": "core-plugins",
-                        "version": "1.5.0-SNAPSHOT",
+                        "version": "1.4.1",
```
Comment (wolf31o2) [inline cdap-ui/templates/apps/predefined/EventClassifier.json]: Keep the version from `develop` not `release/3.6`

File: cdap-ui/templates/apps/predefined/EventClassifier.json [hunk 6]
```diff
@@ -66,7 +66,7 @@
                     "label": "Projection",
                     "artifact": {
                         "name": "core-plugins",
-                        "version": "1.5.0-SNAPSHOT",
+                        "version": "1.4.1",
```
Comment (wolf31o2) [inline cdap-ui/templates/apps/predefined/EventClassifier.json]: Keep the version from `develop` not `release/3.6`

File: cdap-ui/templates/apps/predefined/KafkaToHBase.json [hunk 7]
```diff
@@ -69,7 +69,7 @@
                     "artifact": {
                         "name": "core-plugins",
                         "scope": "SYSTEM",
-                        "version": "1.5.0-SNAPSHOT"
+                        "version": "1.4.1"
```
Comment (wolf31o2) [inline cdap-ui/templates/apps/predefined/KafkaToHBase.json]: Keep the version from `develop` not `release/3.6`

**Review by wolf31o2 — CHANGES_REQUESTED** [review]
All of the JSON files need to keep the version from `develop` not `release/3.6`

**Comment by CuriousVini** [comment]
@wolf31o2 Addressed comments. Built app-templates and cdap locally

**Review by wolf31o2 — APPROVED** [review]
JSON changes and `cdap-examples/SparkPageRank/pom.xml` look good. Were there other conflicts?

**Comment by CuriousVini** [comment]
@wolf31o2 There were conflicts in Hash of build.sh and AuthServerAnnounceTest#startUp() was creating object of NettyRouter which had compilation error. Changed it to use cConf as well.

**Comment by wolf31o2** [comment]
👍 LGTM

**Comment by CuriousVini** [comment]
build is green. merging.

**Pull request merged**